### DISCUSSION TITLE
`Change directory before doing anything` flag

### DISCUSSION THREAD

**Discussion by stefanos82** [c3lang/c3c#1684] [Q&A]
`make`, `cargo`, and others have a `-C` or `--directory` flag to let you `change to DIRECTORY before doing anything`.

In C3, I have to enter the project directory and then execute the command of my choice; for example `c3c build`.

It would have been convenient for me to enter my root directory that contains all my C3 projects and go through them via for loop and execute them one by one: `for dir in *; do c3c -C "$dir" run -q; done`; this way I would test all my projects at once and if one of them fails, will let me know during compile-and-run procedure.

Any feedback on this behavior @lerno ?

**Comment by lerno** [comment]
That's a good idea, will you file an issue?

**Comment by stefanos82** [comment]
Does GitHub give you an option to transfer from Discussions to Issues, like you can do the opposite from Issues to Discussions?

**Comment by stefanos82** [comment]
Nevermind...I had this option on my right :rofl:

**Comment by lerno** [comment]
Ah, yes I just found that one! I'll use it next time.

**Comment by stefanos82** [comment]
Those who are here reading this discussion, please go to #1686
